modules/virt-using-dv-annotations.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-data-volume-annotations.adoc
+
+[id="virt-using-dv-annotations_{context}"]
+= Using data volume annotations
+
+Data volume (DV) annotations allow you to manage pod behavior. You can add one or more allowed annotations to a data volume, which then propagates to the created importer pods.
+
+An example where you can use DV annotations to control the pod’s behavior is istio sidecar injection using the `sidecar.istio.io/inject` annotation. You can use the `sidecar.istio.io/inject` annotation with a setting of `"false"` to disable the automatic istio sidecar injection, preventing errors in the Container Data Importer (CDI) caused by the CDI not inherently recognizing istio.
+
+NOTE: There are multiple methods of using sidecar injection. Istio sidecar injection is used simply for this example.
+
+.Istio sidecar injection disabled example
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: dv-am
+  annotations:
+      sidecar.istio.io/inject: "false" <1>
+spec:
+  source:
+      http:
+         url: "http://example.exampleurl.com"
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+----
+<1> The disabled istio sidecar injection annotation
```

Comment:
Actually after fixing bug https://bugzilla.redhat.com/show_bug.cgi?id=1914833, we already have the default annotation value(false) passing to transfer pods. so maybe we could mentioned that if we don't set the annotation for DV in a namespace with sidecar injection enabled, the annotation will be set to false by default.

Reply:
Hi Yan, in response to this change, I've decided to change the example I'm using in the file. So I'm no longer using the sidecar injection annotation.

What do you think of the file with this change?

Reply:
LGTM. Thanks